pkg/globalmanager/incrementallearningjob.go

```diff
@@ -365,6 +389,16 @@ func (jc *IncrementalJobController) updateIncrementalJobConditions(incrementaljo
 			} else {
 				// watch pod status, if pod running, set type running
 				newConditionType = sednav1.ILJobStageCondRunning
+				pod := jc.getSpecifiedPods(incrementaljob, string(jobStage))
```

Comment:
this getSpecifiedPods can be omited by reusing the previous pod.

Reply:
fix it